packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.d.ts

```diff
@@ -0,0 +1,29 @@
+import { SFC, HTMLProps, ReactNode } from 'react';
+import { OneOf } from '../../typeUtils';
+
+export const ListVariant: {
```

Comment:
Should the name be more unique? Otherwise if there is another ListVariant being exported (e.g. from List package) and the user wants to import { ListVariant } from '@patternfly/react-core' it could spell trouble i think?

Reply:
any suggestions on naming for this? @jschuler